pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
I think this is a typo @ns-rse, suggestion below.
```suggestion
    return x - y
```